test/PolyhedralGeometry/polyhedron.jl

```diff
@@ -621,12 +621,36 @@ end
     @test !Oscar._is_prismic_or_antiprismic(a)
   end
 
+  for s in ("snub_cube", "snub_dodecahedron")
+    ae = eval(Symbol(s))()
+    @test ae isa Polyhedron{<:EmbeddedNumFieldElem}
+    @test !is_platonic_solid(ae)
+    @test is_archimedean_solid(ae)
+    @test !is_johnson_solid(ae)
+    @test is_vertex_transitive(ae)
+    @test Polymake.polytope.isomorphic(Oscar.pm_object(ae), Polymake.polytope.archimedean_solid(s))
+  end
+
+  for s in ("pentagonal_icositetrahedron", "pentagonal_hexecontahedron")
+    ce = eval(Symbol(s))()
```

Comment:
Eval should not be used unless deeply necessary (here it is not)

Reply:
I.e., you can replace
```
for s in ("pentagonal_icositetrahedron", "pentagonal_hexecontahedron")
```
by
```
for s in (pentagonal_icositetrahedron, pentagonal_hexecontahedron)
```